python/cudf/cudf/core/frame.py

```diff
@@ -238,6 +239,44 @@ def _quantiles(
         result._copy_categories(self)
         return result
 
+    def repeat(self, repeats, axis=None):
+
+        assert axis in (None, 0)
```

Comment:
Would be nice to have a comment on why the assertion is here.
Even better, add an assert message with this info.